jni/src/main/java/com/horizen/cswnative/CswFtProverData.java

```diff
@@ -0,0 +1,99 @@
+package com.horizen.cswnative;
+
+import java.util.List;
+
+import com.horizen.fwtnative.FordwardTransferOutput;
+import com.horizen.librustsidechains.Constants;
+import com.horizen.librustsidechains.FieldElement;
+import com.horizen.merkletreenative.MerklePath;
+
+public class CswFtProverData implements AutoCloseable {
+    private final FordwardTransferOutput output;
+    private final byte[] ftInputSecretKey;
+    private final FieldElement mcbScTxsComStart;
+    private final MerklePath merklePathToScHash;
+    private final MerklePath ftTreePath;
+    private final FieldElement scCreationTxHash;
```

Comment:
let's rename it to `scCreationCommitment`.

Reply:
Done